diff --git a/node_modules/@react-three/fiber/dist/events-3afec6fc.cjs.prod.js b/node_modules/@react-three/fiber/dist/events-3afec6fc.cjs.prod.js
index 31cd7a7..17ffd6f 100644
--- a/node_modules/@react-three/fiber/dist/events-3afec6fc.cjs.prod.js
+++ b/node_modules/@react-three/fiber/dist/events-3afec6fc.cjs.prod.js
@@ -1288,7 +1288,7 @@ function createReconciler(config) {
   reconciler.injectIntoDevTools({
     bundleType: typeof process !== 'undefined' &&         "production" !== 'production' ? 1 : 0,
     rendererPackageName: '@react-three/fiber',
-    version: React__namespace.version
+    version: React__namespace.version || '0.0.0'
   });
   return reconciler;
 }
@@ -2476,3 +2476,4 @@ exports.useLoader = useLoader;
 exports.useMutableCallback = useMutableCallback;
 exports.useStore = useStore;
 exports.useThree = useThree;
+
diff --git a/node_modules/@react-three/fiber/dist/events-6ab58ae5.cjs.dev.js b/node_modules/@react-three/fiber/dist/events-6ab58ae5.cjs.dev.js
index 59b365f..0a393e0 100644
--- a/node_modules/@react-three/fiber/dist/events-6ab58ae5.cjs.dev.js
+++ b/node_modules/@react-three/fiber/dist/events-6ab58ae5.cjs.dev.js
@@ -1288,7 +1288,7 @@ function createReconciler(config) {
   reconciler.injectIntoDevTools({
     bundleType: typeof process !== 'undefined' && process.env.NODE_ENV !== 'production' ? 1 : 0,
     rendererPackageName: '@react-three/fiber',
-    version: React__namespace.version
+    version: React__namespace.version || '0.0.0'
   });
   return reconciler;
 }
@@ -2476,3 +2476,4 @@ exports.useLoader = useLoader;
 exports.useMutableCallback = useMutableCallback;
 exports.useStore = useStore;
 exports.useThree = useThree;
+
diff --git a/node_modules/@react-three/fiber/dist/events-f8cd670d.esm.js b/node_modules/@react-three/fiber/dist/events-f8cd670d.esm.js
index a8e8a5e..ddb9f7e 100644
--- a/node_modules/@react-three/fiber/dist/events-f8cd670d.esm.js
+++ b/node_modules/@react-three/fiber/dist/events-f8cd670d.esm.js
@@ -1262,7 +1262,7 @@ function createReconciler(config) {
   reconciler.injectIntoDevTools({
     bundleType: typeof process !== 'undefined' && process.env.NODE_ENV !== 'production' ? 1 : 0,
     rendererPackageName: '@react-three/fiber',
-    version: React.version
+    version: React.version || '0.0.0'
   });
   return reconciler;
 }
@@ -2417,3 +2417,4 @@ function createPointerEvents(store) {
 }
 
 export { useStore as A, Block as B, useThree as C, useFrame as D, ErrorBoundary as E, useGraph as F, useLoader as G, _roots as _, useMutableCallback as a, useIsomorphicLayoutEffect as b, createRoot as c, unmountComponentAtNode as d, extend as e, createPointerEvents as f, createEvents as g, flushGlobalEffects as h, isRef as i, addEffect as j, addAfterEffect as k, addTail as l, invalidate as m, advance as n, createPortal as o, flushSync as p, context as q, reconciler as r, applyProps as s, threeTypes as t, useBridge as u, getRootState as v, dispose as w, act as x, buildGraph as y, useInstanceHandle as z };
